### ISSUE TITLE
Locks are non-functional on Servers

### ISSUE THREAD

**Issue by thephoenixlodge** [TheCodex6824/AuraControl#4]
I've got the config setup to lock crystal gen to the biomes of the Betweenlands dim only. This works perfectly in single player, but on dedicated servers, the crystals are able to spawn anywhere

**Comment by thephoenixlodge** [comment]
A bit of investigation indicates it's probably a load order issue, as you haven't got anything enforcing that your worldgen stuff happens in the right order relative to TC itself

**Comment by TheCodex6824** [comment]
I don't think it's a load order issue, as I explicitly set weights [here](https://github.com/TheCodex6824/AuraControl/blob/master/src/main/java/thecodex6824/auracontrol/AuraControl.java#L97-L98) so that one thing would run before TC, and another after. Of course, there's always the chance some mod is messing with that, but I don't know of any that do.

I also just tried out a minimal dedicated server setup and things seemed to work properly. Can you post the config you're using, and your modlist?

I also apologize for the late response, I've been extremely busy with real life stuff the past few weeks.

**Comment by thephoenixlodge** [comment]
This is part of my modpack Heavens of Sorcery: https://www.curseforge.com/minecraft/modpacks/heavens-of-sorcery
Config is set like this: https://pastebin.com/38vdirLK
Works perfectly in a local singleplayer world, it's just on dedicated servers that the crystals start happily generating everywhere

**Comment by TheCodex6824** [comment]
Sorry for taking a while to get back to you, school's been keeping me busy. I tried another minimal server setup with Betweenlands as well for the heck of it and that worked. I then decided to try out your pack, and I did see crystals everywhere. However, in the server file ("Heavens of Sorcery 0.10.0 Beta-Server 2"), the download does not seem to have AuraControl in the mods folder. The config *is* in the config folder though, so I have a feeling you just left out the jar.
